raiden/transfer/utils.py

```diff
@@ -3,7 +3,30 @@
 from web3 import Web3
 
 from raiden.constants import EMPTY_HASH
+from raiden.storage.sqlite import SQLiteStorage
 from raiden.utils import typing
+from raiden.utils.serialization import serialize_bytes
+
+
+def get_latest_known_balance_proof(
+        storage: SQLiteStorage,
+        balance_hash: typing.BalanceHash,
+):
```

Comment:
add return type: `typing.Optional[BalanceProofSignedState]`

Reply:
Importing `BalanceProofSignedState` from `raiden.transfer.state` will result in circular dependency. So added the "string" type instead.